auth_pass_ip_restriction/models/res_users.py

```diff
@@ -0,0 +1,22 @@
+from odoo import _, models
+from odoo.exceptions import AccessDenied
+from odoo.http import request
+from odoo.tools.config import config
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    def _check_credentials(self, password, env):
+        ignored_endpoints = config.get("auth_pass_ignored_endpoints")
```

Comment:
In particular I would like to know, as a README reader, in what way I should add these multiple endpoints in one configuration line. Are they comma-separated, or Python list-syntax, or...